### DISCUSSION TITLE
Advanced Markers

### DISCUSSION THREAD

**Discussion by lovedeep5** [JustFly1984/react-google-maps-api#3265] [Ideas]
Google previously released Advanced markers, and as per IO 2023 those markers are 60% lighter and easily customizable, how can we implement those with this package?

**Comment by stephent** [comment]
Very good question, given I'm now seeing this in my console:

> As of February 21st, 2024, google.maps.Marker is deprecated. Please use google.maps.marker.AdvancedMarkerElement instead. At this time, google.maps.Marker is not scheduled to be discontinued, but google.maps.marker.AdvancedMarkerElement is recommended over google.maps.Marker. While google.maps.Marker will continue to receive bug fixes for any major regressions, existing bugs in google.maps.Marker will not be addressed. At least 12 months notice will be given before support is discontinued. Please see https://developers.google.com/maps/deprecations for additional details and https://developers.google.com/maps/documentation/javascript/advanced-markers/migration for the migration guide.

**Comment by hasti123** [comment]
any updates on this?
